src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -0,0 +1,123 @@
+package teammates.test.cases.search;
+
+import java.security.InvalidParameterException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackResponseCommentSearchResultBundle;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+import teammates.test.driver.AssertHelper;
+
+/**
+ * SUT: {@link FeedbackResponseCommentsDb},
+ * {@link FeedbackResponseCommentSearchResult}.
+ */
+
+public class FeedbackResponseCommentSearchTest extends BaseSearchTest {
+
+    @Test
+    public void allTests() throws InvalidParameterException {
+
+        FeedbackResponseCommentsDb feedbackResponseCommentsDb = new FeedbackResponseCommentsDb();
+        FeedbackResponseCommentAttributes fdbrI1S1 = dataBundle.feedbackResponseComments
+                .get("comment1FromT1C1ToR1Q1S1C1");
+        FeedbackResponseCommentAttributes fdbrI1S1Q2 = dataBundle.feedbackResponseComments
+                .get("comment1FromT1C1ToR1Q2S1C1");
+        FeedbackResponseCommentAttributes fdbrI1S1Q3 = dataBundle.feedbackResponseComments
+                .get("comment1FromT1C1ToR1Q3S1C1");
+        List<InstructorAttributes> ins1OfCourse1 = Arrays.asList(
+                new InstructorAttributes[] { dataBundle.instructors.get("instructor1OfCourse1") });
+
+        ______TS(
+                "success: search for feedbackresponsecomments; query string does not match any "
+                        + "feedbackresponsecomments; results restricted "
```

Comment:
Prefer 8 spaces indentation. Also, if line doesn't exceed the limit, you don't need to break the string